File: Dailycod/palindrome.py
def isPalindrome(x):
    res=0
    d=x
    if x<0:
        return False
    while x!=0:
        l=x%10
        x=x//10
        res=(res*10)+l
    if res==d:
        return True
    else:
        return False
print(isPalindrome(-121))
        

File: Dailycod/palindrome.c
bool isPalindrome(char* s)
 {
    int i=0;
    int j=strlen(s);
    while(i>j)
    {
        if(!isalnum(s[i]))
        {
            i++;
            
        }
        if(!isalnum(s[j]))
        {
            j--;
        }
        if(tolower(s[i]) != tolower(s[j]))
            return false;
            i++;
            j--;
        
        
    }
return true;

}
